includes/class-accredible-learndash-admin-scripts.php

```diff
@@ -64,24 +64,28 @@ public static function load_page_ajax() {
 				)
 			);
 
-			// phpcs:ignore WordPress.Security.NonceVerification.Recommended
-			if ( isset( $_GET['page'] ) && 'accredible_learndash_auto_issuance' === $_GET['page'] ) {
-				wp_enqueue_script(
-					'accredible-learndash-groups-autocomplete',
-					ACCREDIBLE_LEARNDASH_PLUGIN_URL . 'assets/js/accredible-autocomplete.js',
-					array( 'jquery' ),
-					ACCREDIBLE_LEARNDASH_SCRIPT_VERSION_TOKEN,
-					true
-				);
+			self::enqueue_groups_autocomplete();
```

Comment:
Minor - Don't we need `'accredible_learndash_issuance_list' === $_GET['page']` or something anymore?

Reply:
Yes we do. Updated.